dbschema/progress-summary.esdl

```diff
@@ -0,0 +1,23 @@
+module default {
+  type ProgressSummary {
+    required planned: float32;
+    required actual: float32;
+    
+    required report: ProgressReport; 
+    required period: ProgressSummary::Period;
+    
+    totalVerses: int16;
+    totalVerseEquivalents: int16;
```

Comment:
```suggestion
    totalVerseEquivalents: int32;
```

Reply:
Max 16 is 32k and number of all verses is 31k. Curious why int32 here.

Reply:
Actually `float32`.
Max verse equivalents (for full bible) is 35683.8125)